ghost/ghost/src/common/decorators/permissions.decorator.ts

```diff
@@ -0,0 +1,8 @@
+import {Reflector} from '@nestjs/core';
+
+type UserRole = 'Contributor' | 'Author' | 'Editor' | 'Admin' | 'Owner';
```

Comment:
Maybe add a note here where these values are coming from - the `schema.json` in the legacy codebase. Same for the rest in the file. Just so the future us know where to clean up